packages/devextreme/js/__internal/ui/list/m_list.edit.decorator.selection.ts

```diff
@@ -74,6 +76,9 @@ class EditDecoratorSelection extends EditDecorator {
         const isUiClick = !!event;
         if (isUiClick) {
           component._valueChangeEventInstance = undefined;
+          if (this._controlClass === SELECT_RADIO_BUTTON_CLASS && value === false) {
```

Comment:
I see why it works but the code looks to specific to me.
Isn't it better to restore `_valueChangeEventInstance` in RadioButton's level and value change call?
Then we'll not have this specific condition, we'll be sure it'd be not broken in another places and radiobutton's algorithm will be more stable itself